app/pages/sections/order-items/associated-services/order-item/contextCreator.test.js

```diff
@@ -0,0 +1,137 @@
+import commonManifest from './commonManifest.json';
+import flatDeclarativeManifest from './flat/declarative/manifest.json';
+import { getContext } from './contextCreator';
+
+describe('associated-services order-item contextCreator', () => {
+  describe('getContext', () => {
+    it('should return the backLinkText', () => {
+      const context = getContext({
+        commonManifest,
+      });
+      expect(context.backLinkText).toEqual(commonManifest.backLinkText);
+    });
+
+    it('should return the backLinkHref to associated-services when order item id is not neworderitem', () => {
+      const context = getContext({
+        commonManifest,
+        orderId: 'order-1',
+      });
+      expect(context.backLinkHref).toEqual('/order/organisation/order-1/associated-services');
+    });
+
+    it('should return the backLinkHref to recipient when order item id is neworderitem', () => {

```

Comment:
We need to change,... from `should return the backLinkHref to recipient ` to `should return the backLinkHref to the select a price page`